learning_resources/constants.py

```diff
@@ -185,3 +185,47 @@ class LearningResourceRelationTypes(TextChoices):
     "STS": "Science, Technology, and Society",
     "WGS": "Women's and Gender Studies",
 }
+
+
+LEARNING_RESOURCE_SORTBY_OPTIONS = {
```

Comment:
Can you make these changes to the learning_resource search endpoint as well. Also does it make sense for both endpoints to have the same sort options?